src/Raven.Client/Documents/Subscriptions/SubscriptionCriteria.cs

```diff
@@ -5,13 +5,30 @@
 // -----------------------------------------------------------------------
 
 using System;
+using Raven.Client.Documents.Replication;
+using Raven.Client.Documents.Replication.Messages;
 using Sparrow.Json;
 
 namespace Raven.Client.Documents.Subscriptions
 {
+    public class SubscriptionRaftState:IDatabaseTask
+    {
+        public SubscriptionRaftState()
+        {
+
+        }
+        public SubscriptionCriteria Criteria { get; set; }
+        public ChangeVectorEntry[] ChangeVector { get; set; }
+        public long Etag { get; set; }
```

Comment:
Yeah, but if I'll do that, we'll have a consistency problem.. in Indexes and Transformers we call the identification Etag now. Note that it's not the "SubscriptionCriteria" obejct that the client can see, but an inner state.